assets/src/styles/blocks/OldCovers/TakeActionBoxoutEditor.scss

```diff
@@ -0,0 +1,20 @@
+.cover-card.action-card {
+  position: relative;
+  max-width: 33%;
+  margin-right: 0;
+  margin-left: auto;
+
+  a {
+    pointer-events: none;
+  }
+
+  html[dir="rtl"] & {
+    margin-left: 0;
+    margin-right: auto;
+  }
+
+  .cover-card-excerpt,
+  .cover-card-btn {
+    display: block;
```

Comment:
Just checking, are you keeping this behavior difference from the frontend on purpose ?